### PR TITLE
create layout L011 in  commercial management section

### PR DESCRIPTION
**Pull request by Andresbl123**
New props are implemented in the `<TableBoard />` component, allowing the appearance of the headings to be adjusted, and the layout of the commercial management section is implemented.

### CODE REVIEW

**Review by novadeveloper14 — APPROVED** [review]
the development meets the requirements

File: src/components/data/Accordeon/index.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+import { StyledDetails, StyledSummary, StyledCollapseIcon } from "./styles";
+
+export interface IAccordionProps {
+  name: string;
+  title: React.ReactNode;
+  content?: React.ReactNode;
+}
+
+export const Accordeon = (props: IAccordionProps) => {
```
Comment (carin91) [inline src/components/data/Accordeon/index.tsx:13]: Adjust the spelling for the component name: Accordion

File: src/pages/board/outlets/financialReporting/CommercialManagement/index.tsx [hunk 2]
```diff
@@ -117,7 +118,7 @@ export const ComercialManagement = (props: ComercialManagementProps) => {
         </Stack>
         {collapse && <StyledDivider />}
         {collapse && (
-          <Stack>
+          <Stack direction="column" gap="10px">
```
Comment (carin91) [inline src/pages/board/outlets/financialReporting/CommercialManagement/index.tsx:121]: change the gap value with inube.spacing.s250 or equivalent

**Review by carin91 — APPROVED** [review]

File: src/components/data/Accordeon/index.tsx (hunk 1 above)
Comment (Andresbl123) [reply src/components/data/Accordeon/index.tsx:13]: resuelto

File: src/pages/board/outlets/financialReporting/CommercialManagement/index.tsx (hunk 2 above)
Comment (Andresbl123) [reply src/pages/board/outlets/financialReporting/CommercialManagement/index.tsx:121]: resuelto

**Pull request merged**